rosdep/python.yaml

```diff
@@ -2855,6 +2855,12 @@ python-path.py:
   ubuntu:
     pip:
       packages: [path.py]
+python-pathlib:
+  arch: [python2-pathlib]
+  debian: [python-pathlib]
+  fedora: [python-pathlib]
```

Comment:
As of Fedora 31, python-pathlib is no longer available.  This is because of the larger effort to remove python2 from the Fedora default install.  Given Fedora's typical [support schedule](https://en.wikipedia.org/wiki/Fedora_version_history), I'd expect that Fedora 30 is going out of support by the end of May.  Because of that, I'd suggest that we just remove the fedora entry here, as it won't be valid in 2 more months.

Reply:
Done 112e671